pkg/features/kube_features.go

```diff
@@ -325,6 +325,14 @@ const (
 	// beta:  v1.18
 	// GA:    v1.20
 	//
+	// Enables the endPort field in NetworkPolicy to enable a Port Range behavior in Network Policies.
+	NetworkPolicyEndPort featuregate.Feature = "NetworkPolicyEndPort"
+
+	// owner: @rikatz
+	// alpha: v1.20
+	// beta:  v1.21
+	// GA:    v1.22
+	//
 	// Enables SCTP as new protocol for Service ports, NetworkPolicy, and ContainerPort in Pod/Containers definition
```

Comment:
This got inserted wrong; you added `NetworkPolicyEndPort` after the `SCTPEnabled` alpha/beta/GA definitions, and then changed `SCTPEnabled` to have `NetworkPolicyEndPort`'s

Reply:
+1

Reply:
The old but gold copy/paste :man_facepalming: 

Corrected